docs/installation/configuration/opennotifs_config_cli.rst

```diff
@@ -21,96 +18,57 @@ You can get the full command documentation with:
    running the command and you then run the exact same command again, the manual
    changes will be reverted.
 
-
 Preparation
 ===========
 
 The command executes the list of pluggable configuration steps, and each step
-requires specific environment variables, that should be prepared.
-Here is the description of all available configuration steps and the environment variables,
-use by each step.
-
-Sites configuration
--------------------
+requires specific configuration information, that should be prepared.
+Here is the description of all available configuration steps and the shape of the data,
+used by each step.
 
-Configure the domain where Open Notificaties is hosted
 
-* ``SITES_CONFIG_ENABLE``: enable Site configuration. Defaults to ``False``.
-* ``OPENNOTIFICATIES_DOMAIN``:  a ``[host]:[port]`` or ``[host]`` value. Required.
-* ``OPENNOTIFICATIES_ORGANIZATION``: name of Open Notificaties organization. Required.
+Services configuration
+----------------------
 
-Authorization configuration
----------------------------
+TODO: add generated documentation for ``zgw_consumers.ServiceConfigurationStep``
```

Comment:
Are these `TODO`'s to be generated later on?

Reply:
Yeah I wanted to add these placeholders to make sure we don't forget about it, but this will probably happen in january next year